install/websocket/docs/index.html

```diff
@@ -1,25 +1,7 @@
 <!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
 <html>
 <head>
-<!--
```

Comment:
Please keep the license header. Update the 2nd date in copyright from 2019 to 2020.